modules/index.js

```diff
@@ -14,26 +14,73 @@
 import { KNOWN_PROPERTIES, DEFAULT_TRACKING_EVENTS } from './defaults.js';
 import { urlSanitizers } from './utils.js';
 import { targetSelector, sourceSelector } from './dom.js';
-import {
-  addAdsParametersTracking,
-  addCookieConsentTracking,
-  addEmailParameterTracking,
-  addUTMParametersTracking,
-} from './martech.js';
 import { fflags } from './fflags.js';
 
 const { sampleRUM, queue, isSelected } = (window.hlx && window.hlx.rum) ? window.hlx.rum
   /* c8 ignore next */ : {};
 
+const createMO = (cb) => (window.MutationObserver ? new MutationObserver(cb)
+/* c8 ignore next */ : {});
+
 // blocks mutation observer
-// eslint-disable-next-line no-use-before-define, max-len
-const blocksMO = window.MutationObserver ? new MutationObserver(blocksMCB)
-  /* c8 ignore next */ : {};
+// eslint-disable-next-line no-use-before-define
+const blocksMO = createMO(blocksMCB);
 
 // media mutation observer
-// eslint-disable-next-line no-use-before-define, max-len
-const mediaMO = window.MutationObserver ? new MutationObserver(mediaMCB)
-  /* c8 ignore next */ : {};
+// eslint-disable-next-line no-use-before-define
+const mediaMO = createMO(mediaMCB);
+
+// Check for the presence of URL parameters
+const hasUrlParameters = ({ urlParameters }) => urlParameters.keys().length > 0;
+// Check for the presence of a given cookie
+const hasCookieKey = (key) => () => document.cookie.split(';').map((c) => c.trim()).some((cookie) => cookie.startsWith(`${key}=`));
+
+const pluginBasePath = new URL('.rum/@adobe/helix-rum-enhancer@^2/src/plugins', sampleRUM.baseURL).href;
+
+const PLUGINS = {
+  cwv: `${pluginBasePath}/cwv.js`,
+  navigation: `${pluginBasePath}/navigation.js`,
+  // Interactive elements
+  form: { url: `${pluginBasePath}/form.js`, condition: () => document.body.querySelector('form') },
```

Comment:
```suggestion
  form: () => document.body.querySelector('form'),
```

We can do more with conventions here, I'd say.